src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -1253,6 +1253,7 @@ def do_showall(self, args):
     client = SystemAdministratorIntegrator()
     silentHosts = client.getSilentHosts()
     respondingHosts = client.getRespondingHosts()
+    allHosts = len(silentHosts) + len(respondingHosts)
```

Comment:
I would expect a variable named `allHosts` to contain a list of all hosts, how about:
```suggestion
    totalHosts = len(silentHosts) + len(respondingHosts)
```